api/models/Post.js

```diff
@@ -512,10 +552,25 @@ module.exports = bookshelf.Model.extend(Object.assign({
     EVENT: 'event',
     OFFER: 'offer',
     PROJECT: 'project',
+    PROPOSAL: 'proposal',
     REQUEST: 'request',
     RESOURCE: 'resource',
     THREAD: 'thread',
-    WELCOME: 'welcome',
+    WELCOME: 'welcome'
+  },
+
+  Proposal_Status: {
```

Comment:
oh hmm, I guess Type is not capitalized 🤷‍♂️ I guess leave it this way then

Reply:
Yeah, thats what I was going off